### PR TITLE
Bump dep on semver to 1.0.0

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

**Review by dfreese — APPROVED** [review]
LGTM.  If you don't mind, I might wait on #425.  I had a minor comment there, and then I think it's good to go.

**Comment by dfreese** [comment]
@illicitonion sorry for the delay on that, but #425 has been merged.

File: impl/src/metadata.rs [hunk 1]
```diff
@@ -516,9 +515,7 @@ pub mod tests {
 
   impl DummyCargoMetadataFetcher {
     fn render_metadata(&self, mock_workspace_path: &Path) -> Option<Metadata> {
-      if self.metadata_template.is_none() {
-        return None;
-      }
+      self.metadata_template.as_ref()?;
```
Comment (dfreese) [inline impl/src/metadata.rs:518]: Can you store this to a value and use it later on line 532?  That'll remove both the unwrap in those lines, and make the intent of this more clear.

**Review by dfreese — APPROVED** [review]
Looks like it's breaking the bazel build of this by not having serde-derive as a proc macro dependency?  I'm not super familiar with the bazel build for this, but does something need to be regenerated?  @UebelAndre, not sure if you'd see an obvious thing right away here.

Overall, looks good to me, though.

**Comment by UebelAndre** [comment]
Yes, if the dependencies are updated, then `cargo-raze` will need to be re-bootstrapped
```
bazel run //tools:bootstrap
```
Then commit the results and things should be good.

PS: @dfreese Can you check slack?

**Comment by illicitonion** [comment]
Yay, rebasing past https://github.com/google/cargo-raze/pull/441 fixed CI :)

**Pull request merged**